src/commands/dev/dev.js

```diff
@@ -335,7 +335,13 @@ const createDevCommand = (program) => {
         'specify the path to a local GeoIP location database in MMDB format',
       ).hideHelp(),
     )
-    .addExamples(['netlify dev', 'netlify dev -d public', 'netlify dev -c "hugo server -w" --targetPort 1313'])
+    .addExamples([
+      'netlify dev',
+      'netlify dev -d public',
+      'netlify dev -c "hugo server -w" --targetPort 1313',
+      'BROWSER=firefox netlify dev # auto open firefox instead of preferred browser',
```

Comment:
This bit should work (setting to firefox) should work, but currently doesn't due to an upstream bug:
https://github.com/michaellzc/better-opn/pull/38

Reply:
You're right, better to remove it for now